src/pipecat/services/google/tts.py

```diff
@@ -379,3 +379,148 @@ async def run_tts(self, text: str) -> AsyncGenerator[Frame, None]:
             logger.exception(f"{self} error generating TTS: {e}")
             error_message = f"TTS generation error: {str(e)}"
             yield ErrorFrame(error=error_message)
+
+
+class GoogleTTSService(TTSService):
+    class InputParams(BaseModel):
+        pitch: Optional[str] = None
+        rate: Optional[str] = None
+        volume: Optional[str] = None
+        emphasis: Optional[Literal["strong", "moderate", "reduced", "none"]] = None
+        language: Optional[Language] = Language.EN
+        gender: Optional[Literal["male", "female", "neutral"]] = None
+        google_style: Optional[Literal["apologetic", "calm", "empathetic", "firm", "lively"]] = None
+
+    def __init__(
+        self,
+        *,
+        credentials: Optional[str] = None,
+        credentials_path: Optional[str] = None,
+        voice_id: str = "en-US-Neural2-A",
```

Comment:
The Chirp 3 voices are not fully supported. Let's make it the default here in and in the HttpService too.

```suggestion
        voice_id: str = "en-US-Chirp3-HD-Charon",
```

Reply:
Ahh, looks like only Chirp 3 voices are supported. In trying a non-Chirp 3, I get:
```
error: TTS generation error: 400 Currently, only Chirp 3: HD voices are supported for streaming synthesis., fatal: False)
```

Reply:
@aristid can you update the default voice here to `en-US-Chirp3-HD-Charon`?